examples/notebooks/Intro_to_Quickvision_with_W&B.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>Just add support for various pretrained weights.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Quick-AI/quickvision/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>